### PR TITLE
[RN][Blog] Add blogpost for 0.75 

### PR DESCRIPTION
**Pull request by cipolleschi**
We are about to release version 0.75 of react Native.
This will be the blogpost for it.

### CODE REVIEW

File: website/blog/2024-08-12-release-0.75.md [hunk 1]
```diff
@@ -0,0 +1,487 @@
...
+
+One notable and highly requested feature is the support for % in various places, such as `gaps` and `translation`
+
+:::warning
+These features are available only for the New Architecture. If you are keen to use them, please consider migrating to it.
+:::
+
+#### % in Gaps
+
+With 0.75, the `gap`, `clomumnGap` and `rowGap`props described [here](https://reactnative.dev/docs/flexbox#row-gap-column-gap-and-gap) now support a string with a % as possible value.
```
Comment (frankcalise) [inline website/blog/2024-08-12-release-0.75.md:107]:
```suggestion
With 0.75, the `gap`, `columnGap` and `rowGap`props described [here](https://reactnative.dev/docs/flexbox#row-gap-column-gap-and-gap) now support a string with a % as possible value.
```

**Review by frankcalise — COMMENTED** [review]
Fixed a typo

File: website/blog/2024-08-12-release-0.75.md [hunk 2]
```diff
@@ -0,0 +1,487 @@
...
+
+:::info
+In order to offer a smoother migration experience, the `react-native@0.75.0` package is still depending on `@react-native-community/cli` but we’re planning on removing this dependency in a future version of React Native.
+:::
+
+#### Auto-linking performance improvements
+
+During this work in updating the `init` command, we also spent time rewriting the auto-linking logic to be more performant. This results in faster build speed for both Android and iOS.
+
+With React Native 0.75, the auto-linking step is now running twice as fast on iOS and ~9 times faster on Android. You can read more about these improvements in this dedicated post [here](https://github.com/react-native-community/discussions-and-proposals/discussions)
```
Comment (Kudo) [inline website/blog/2024-08-12-release-0.75.md:91]:
```suggestion
With React Native 0.75, if you are using Expo, the auto-linking step could now be running ~1.5 as fast on iOS and ~6.5 times faster on Android. You can read more about these improvements in this dedicated post [here](https://github.com/react-native-community/discussions-and-proposals/discussions/814)
```

trying to average the faster time from my benchmark.
(1.35 + 1.64) / 2 ~= 1.5
(2.88 + 10.24) / 2 ~= 6.5

File: website/blog/2024-08-12-release-0.75.md [hunk 3]
```diff
@@ -0,0 +1,487 @@
...
+```objc
+  + (void)didReceiveLocalNotification:(UILocalNotification *)notification;
+  + (void)didReceiveRemoteNotification:(NSDictionary *)notification;
+```
+
+Instead, use `didReceiveNotification:(UNNotification *)notification`.
+
+#### Community CLI: Removal of ram-bundle and profile-hermes commands
+
+We would like two significant removals coming from the Community CLI: the commands `ram-bundle` and `profile-hermes` have been removed.
```
Comment (j-piasecki) [inline website/blog/2024-08-12-release-0.75.md:397]:
```suggestion
We would like to announce two significant removals coming from the Community CLI: the commands `ram-bundle` and `profile-hermes` have been removed.
```

Not sure if that is the intended wording, but something is definitely missing in this sentence.

Comment (cortinico) [reply website/blog/2024-08-12-release-0.75.md:397]: yup thank you

File: website/blog/2024-08-12-release-0.75.md [hunk 4]
```diff
@@ -0,0 +1,490 @@
...
+
+Since we last shipped Yoga [version 3.0](https://reactnative.dev/blog/2024/04/22/release-0.74#yoga-30) in React Native 0.74, we kept on pushing many improvements and new layout capabilities for your applications. React Native 0.75 ships with Yoga 3.1 and you can learn more about what’s new in the official Yoga’s [release blog post](https://www.yogalayout.dev/blog/announcing-yoga-3.1).
+
+One notable and highly requested feature is the support for `%` values in various places, such as `gaps` and `translation`
+
+:::info
+These features are available only for the New Architecture. If you are keen to use them, please consider migrating to it.
+:::
+
+#### `%` Values in Gaps
```
Comment (Simek) [inline website/blog/2024-08-12-release-0.75.md:49]:
```suggestion
#### Percentage Values in Gaps
```

Percent character looks a bit off at the beginning of headers, maybe we should use the word-based alternative?

File: website/blog/2024-08-12-release-0.75.md [hunk 5]
```diff
@@ -0,0 +1,490 @@
...
+}
+```
+
+Will be rendered as follows:
+
+| Android                                                      | iOS                                                  |
+| ------------------------------------------------------------ | ---------------------------------------------------- |
+| ![Android Gaps](../static/blog/assets/0.75-android-gaps.png) | ![iOS Gaps](../static/blog/assets/0.75-ios-gaps.png) |
+
+#### `%` values in Translation
```
Comment (Simek) [inline website/blog/2024-08-12-release-0.75.md:148]:
```suggestion
#### Percentage Values in Translation
```

Same case as above.

File: website/blog/2024-08-12-release-0.75.md [hunk 6]
```diff
@@ -0,0 +1,490 @@
...
+
+One notable and highly requested feature is the support for `%` values in various places, such as `gaps` and `translation`
+
+:::info
+These features are available only for the New Architecture. If you are keen to use them, please consider migrating to it.
+:::
+
+#### Percentage Values in Gaps
+
+With 0.75, the `gap`, `columnGap` and `rowGap`props described [here](https://reactnative.dev/docs/flexbox#row-gap-column-gap-and-gap) now support a string with a `%` value as possible value.
```
Comment (TheSavior) [inline website/blog/2024-08-12-release-0.75.md:51]: missing a space

**Pull request merged**